Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -619,6 +619,10 @@ final class StatusTableViewController: ChartsTableViewController {
 
         let hudIsVisible = self.shouldShowHUD
         let statusIsVisible = self.shouldShowStatus
+        
+        if hudIsVisible != hudView?.glucoseHUD.isVisible {
```

Comment:
[nit] Why not just set the `isVisible` property value and let the property's `didSet` determine if it changed and take appropriate action?